src/wallet/db.h

```diff
@@ -214,6 +214,7 @@ enum class DatabaseStatus {
     FAILED_CREATE,
     FAILED_VERIFY,
     FAILED_ENCRYPT,
+    FAILED,
```

Comment:
In commit "bugfix: make LoadWallet assigns status always" (f32ddd81267cfca9f3c07006cef7c80f79a48527)

Can `FAILED` be renamed to `FAILED_LOAD` and moved immediately after `FAILED_CREATE` above? `FAILED_LOAD` and `FAILED_CREATE` would be the most generic failures, but it'd be good to distinguish between loading an existing wallet and creating a new one.